src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        for (Ball ball: balls) {
+            ball = Lottery.getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
you don't need an array to hold the `Ball` instances - your goal is just to print them

apart from that you need to declare a constant for the magic number `3`

don't forget that `getRandomBall()` should not be static, so you need to create an instance of `Lottery` class

after that you can use a `for` loop and just print the result of `getRandomBall()` method

```suggestion
        Lottery lottery = new Lottery():
        for (int i =0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```